modules/standard/CommDiagnostics.chpl

```diff
@@ -306,7 +315,7 @@ module CommDiagnostics
     Start counting communication operations across the whole program.
    */
   proc startCommDiagnostics() {
-    chpl_comm_startDiagnostics(commDiagsPrintUnstable);
+    chpl_comm_startDiagnostics(commDiagsStacktrace, commDiagsPrintUnstable);
```

Comment:
`startCommDiagnostics()` and `startCommDiagnosticsHere()` shouldn't need to send the `commDiagsStackTrace` value down to the runtime support code.  The latter only uses it if verbose comm is turned on by `startVerboseComm()` or `startVerboseCommHere()`, and those two already send it to the runtime themselves.